src/list.tsx

```diff
@@ -242,7 +218,34 @@ const Content: React.FC<ContentProps> = ({
                 </Stack>
             </div>
             {renderBadge()}
-            {renderRight()}
+            {type === 'chevron' && (
+                <Box paddingLeft={16} className={classNames(classes.center, {[classes.disabled]: disabled})}>
+                    <IconChevron
+                        color={isInverse ? colors.inverse : colors.neutralMedium}
+                        direction="right"
+                    />
+                </Box>
+            )}
+            {type === 'control' && <div className={classes.right}>{right?.({centerY})}</div>}
+            {type === 'custom' && (
+                <>
+                    <div className={classNames(classes.right, {[classes.disabled]: disabled})}>
+                        {right?.({centerY})}
+                    </div>
+                    {navigable && (
+                        <div className={classNames(classes.right, {[classes.disabled]: disabled})}>
+                            <div
+                                style={centerY ? {display: 'flex', alignItems: 'center', height: '100%'} : {}}
+                            >
+                                <IconChevron
+                                    color={isInverse ? colors.inverse : colors.neutralMedium}
+                                    direction="right"
+                                />
+                            </div>
```

Comment:
IIRC, the chevron should be *always* centered vertically, is that ok @yceballost?
If that's the case, then this should be inconditionally centered in the vertical:
```suggestion
                            <div
                                style={{display: 'flex', alignItems: 'center', height: '100%'}}
                            >
                                <IconChevron
                                    color={isInverse ? colors.inverse : colors.neutralMedium}
                                    direction="right"
                                />
                            </div>
```

Reply:
true, chevron always has to be placed centered